redaxo/src/core/lib/api_function.php

```diff
@@ -90,6 +90,15 @@ public static function factory()
         return null;
     }
 
+    public static function getUrlParams()
+    {
+        $class = get_called_class();
```

Comment:
exception wenn class == this-class ?

Reply:
Jupp, wollte bloß erst das grundsätzliche klären, dann der Feinschliff ;)